src/newLaunch/lbp/config.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  ILaunchConfig,
+  IGeneral,
+  IProjectDetails,
+  ITokenDetails,
+  IContactDetails,
+} from "newLaunch/launchConfig";
+
+export interface ILbpDetails {
+  fundingTokenAddress: string,
+  amountProjectToken: string,
+  amountFundingToken: string,
+  startDate: string,
```

Comment:
I propose that this and other props here should be in a ILaunchDetails interface that should be part of the shared config interface.

Seed and Lbp config interfaces I think can override the `launchDetails` property like, for example, `launchDetails: ISeedDetails` where `ISeedDetails implements ILaunchDetails`